src/helpers/required.ts

```diff
@@ -1,18 +1,20 @@
-// Copyright (c) Consensys Software Inc. All rights reserved.
+// Copyright (c) 2022. Consensys Software Inc. All rights reserved.
 // Licensed under the MIT license.
 
+import {AppTypes, Constants, OptionalApps, RequiredApps} from '@/Constants';
 import {getTruffleConfigUri, TruffleConfig} from '@/helpers/TruffleConfiguration';
+import {AbstractWorkspaceManager, getPathByPlatform, getWorkspaceRoot} from '@/helpers/workspace';
+import {createOutputInst, OutputLabel} from '@/Output';
+import {Telemetry} from '@/TelemetryClient';
 import fs from 'fs-extra';
 import path from 'path';
 import semver from 'semver';
 import {commands, ProgressLocation, window} from 'vscode';
-import {Constants, RequiredApps} from '@/Constants';
-import {getWorkspaceRoot} from '@/helpers/workspace';
-import {Output, OutputLabel} from '@/Output';
-import {Telemetry} from '@/TelemetryClient';
 import {executeCommand, tryExecuteCommand} from './command';
 
 export namespace required {
+  const output = createOutputInst(OutputLabel.requirements);
```

Comment:
This is your logging code wrapped in a function, you just need to change that impl under the hood for Output and this would still work unless thats a complete rewrite. from what I saw it looks like some minor functional changes.

Reply:
I can jus remove the methods that have been refactored out.